src/WindowsAzureMessaging/WindowsAzureMessaging/Models/MSNotificationHubMessage.m

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSNotificationHubMessage.h"
+
+@implementation MSNotificationHubMessage
+
+@synthesize title;
```

Comment:
Basically synthesize is options, it all depends how you wanna see your internal properties to look like. But anyway, please refactor it to one line, like @synthesize title, message, badge, additionalData;

Reply:
When u use multi-line, it makes following changes diff cleaner